source/jormungandr/jormungandr/parking_space_availability/bss/stands.py

```diff
@@ -27,17 +27,22 @@
 # https://groups.google.com/d/forum/navitia
 # www.navitia.io
 from __future__ import absolute_import, print_function, unicode_literals, division
+from enum import Enum
 
-status_list = {'OPEN': 'open', 'CLOSED': 'closed', 'UNAVAILABLE': 'unavailable'}
-
+class StandsStatus(Enum):
+    unavailable = -1
+    closed = 0
+    open = 1
 
 class Stands(object):
 
     def __init__(self, available_places, available_bikes, status=None):
+        if status is not None and not isinstance(status, StandsStatus):
+            raise TypeError('status must be a StandsStatus enum')
         self.available_places = available_places
         self.available_bikes = available_bikes
         self.total_stands = available_places + available_bikes
-        self.status = status_list.get(status)
+        self.status = status.name
```

Comment:
For now I get an recursion error if I store the actual enum...

Reply:
you mean if you store the enum in the status_list? or store the enum as an attribute of `Stands`?

Reply:
recursion caused by serpy?

Reply:
If I just store the enum as an attribute, even if I do nothing with it (like add `self.toto = status`).
Then a simple request to a POI with stand breaks... To be discussed/looked into.